src/entities/customer-info.ts

```diff
@@ -111,6 +129,113 @@ export interface EntitlementInfos {
   readonly active: { [entitlementId: string]: EntitlementInfo };
 }
 
+/**
+ * Information that represents a non-subscription purchase made by a user.
+ * @public
+ */
+export interface NonSubscriptionTransaction {
+  /**
+   * The unique identifier for the transaction created by RevenueCat.
+   */
+  readonly transactionIdentifier: string;
+  /**
+   * The product identifier.
+   */
+  readonly productIdentifier: string;
+  /**
+   * The date that the store charged the user’s account.
+   */
+  readonly purchaseDate: Date;
+  /**
+   * The unique identifier for the transaction created by the Store.
+   */
+  readonly storeTransactionId: string | null;
+  /**
+   * The {@link Store} where the transaction was made.
+   */
+  readonly store: Store;
+}
+
+/**
+ * Subscription purchases of the Customer.
+ * @public
+ */
+export interface SubscriptionInfo {
+  /**
+   * The product identifier.
+   */
+  readonly productIdentifier: string;
+  /**
+   * Date when the last subscription period started.
+   */
+  readonly purchaseDate: Date;
+  /**
+   * Date when this subscription first started. This property does not update with renewals.
+   * This property also does not update for product changes within a subscription group or
+   * re-subscriptions by lapsed subscribers.
+   */
+  readonly originalPurchaseDate: Date | null;
+  /**
+   * Date when the subscription expires/expired
+   */
+  readonly expiresDate: Date | null;
+  /**
+   * Store where the subscription was purchased.
+   */
+  readonly store: Store;
+  /**
+   * Date when RevenueCat detected that auto-renewal was turned off for this subscription.
+   * Note the subscription may still be active, check the {@link SubscriptionInfo.expiresDate} attribute.
+   */
+  readonly unsubscribeDetectedAt: Date | null;
+  /**
+   * Whether or not the purchase was made in sandbox mode.
+   */
+  readonly isSandbox: boolean;
+  /**
+   * Date when RevenueCat detected any billing issues with this subscription.
+   * If and when the billing issue gets resolved, this field is set to null.
+   * Note the subscription may still be active, check the {@link SubscriptionInfo.expiresDate} attribute.
+   */
+  readonly billingIssuesDetectedAt: Date | null;
+  /**
+   * Date when any grace period for this subscription expires/expired.
+   * null if the customer has never been in a grace period.
+   */
+  readonly gracePeriodExpiresDate: Date | null;
+  /**
+   * How the Customer received access to this subscription:
+   * - "PURCHASED": The customer bought the subscription.
+   * - "FAMILY_SHARED": The Customer has access to the product via their family.
```

Comment:
nit, for consistency
```suggestion
   * - "FAMILY_SHARED": The customer has access to the product via their family.
```